grafana/values.yaml

```diff
@@ -0,0 +1,26 @@
+clusterName: "" # @schema description: Name of the cluster to configure the platform Grafana for. This value is required.
```

Comment:
The sschema should enforce required values. Here's an example of how https://github.com/losisin/helm-values-schema-json/blob/main/testdata/full.yaml#L3